CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing
+1. Fork this repository to your own account
+2. Make your changes and verify that tests pass
```

Comment:
Pls add specific commands on how to verify tests

Reply:
`save-cloud:check` task, isn't it?

Reply:
No, it should be something like `./gradlew check` or even `./gradlew build`

Reply:
ok, will do it